modules/msk/snppileup/main.nf

```diff
@@ -0,0 +1,56 @@
+process SNPPILEUP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://mskcc/facets-suite:2.0.9':
+        'docker.io/mskcc/facets-suite:2.0.9' }"
+
+    input:
+
+    tuple val(meta),  path(normal), path(normal_index)
+    tuple val(meta1), path(tumor),  path(tumor_index)
+    tuple val(meta2), path(dbsnp),  path(dbsnp_index)
+
+
+    output:
+    tuple val(meta), path("*.snp_pileup.gz")   , emit: pileup
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    /usr/bin/snp-pileup \
+        ${args} \
+        ${dbsnp} \
+        ${prefix}.snp_pileup.gz \
+        ${normal} \
+        ${tumor}
```

Comment:
do any optional args need to be specified positionally after the required file arguments? If so is adding an `args2` necessary?

Reply:
There is no optional args after file arguments, so there is no need for an args2